components/inputs/input-color.js

```diff
@@ -285,6 +285,10 @@ class InputColor extends FocusMixin(FormElementMixin(LocalizeCoreElement(LitElem
 
 		if (changedProperties.has('label') || changedProperties.has('type')) this._validateLabel();
 
+		if (changedProperties.has('value') || changedProperties.has('type') || changedProperties.has('disallowNone')) {
```

Comment:
The value is a function of these 3 properties, so anytime they change we know to update the form value.